docs/process/ci_build.md

```diff
@@ -102,33 +111,41 @@ Less frequently, there might be specific changes that cannot be tested via a PR
-* Benchmarking
-  - We use the [rhysd/github-action-benchmark](https://github.com/rhysd/github-action-benchmark). It does the following
-    1. For a handful of programming languages (including Rust), it runs the most common benchmark tool
-    1. It converts the benchmark tool's output into a JSON file
-    1. It stores the JSON file in a separate branch. It produces a static HTML that renders the JSON file as a dashboard.
-    1. It commits the output JSON + HTML as a commit in a separate branch. This preserves the data for future invocations, when conversion of benchmark output to JSON is appended to the previous JSON to create the new JSON data file.
-  - We copy the benchmark dashboards along with the API docs in `unicode-org/icu4x-docs`. This allows the GH Pages for the main repo to be used for general purposes, if needed (ex: landing page, blog, etc.).
-* Run CI checks locally
-  - Developers [need an easy way to run the same commands and options locally as they are run in CI](https://github.com/unicode-org/icu4x/issues/334)
-  - Currently, there is no official way to invoke a Github Actions workflow locally
-  - Since we run tests on all major OS types, we need a cross-platform solution to running commands
-  - Rust's `cargo-makefile` is a plugin for the `cargo` build tool that allows creation of a sequence of commands and options using an alias
-  - Using `cargo-makefile`, the CI commands were rewritten to use the new aliases
-  - `cargo-makefile` has built-in support for the inline inclusion of scripts written in [Duckscript](https://github.com/sagiegurari/duckscript)
-    * [Example of an inline script in a `cargo-makefile` task](https://github.com/unicode-org/icu4x/pull/381/files#diff-9375fd04332c86472d7be397ef09428cb86babd8826880a5835bd1d1c1bdbc08R23-R48) that ensures that all new source files have a specific license header
-* Code coverage
-  - Code coverage is configured in a separate workflow since it is completely independent of the operations for build, test, format/style-checking, linting/error-prone-checking, API docs, benchmarking.
-  - Two code coverage dashboard services exist:
-    * [Codecov](https://codecov.io/)
-      - [config for Codecov](https://github.com/unicode-org/icu4x/commit/fd524a91df3f5aef2efb908adddd051827972328#diff-a2115d277b5ca5a2f09a999e53440839cf332b94da177f3d1766334555b0f7c6R49-R53)
-    * [Coveralls](https://coveralls.io/)
-      - [config for Coveralls](https://github.com/unicode-org/icu4x/commit/00c514f3a189787c4e77704eb98714e6b6430d37#diff-a2115d277b5ca5a2f09a999e53440839cf332b94da177f3d1766334555b0f7c6R36-R40)
-  - Both code coverage dashboard services automatically support most output file formats for most programming languages' code coverage tools
\ No newline at end of file
+## Examples of Various CI Functionality
+
```

Comment:
Suggestion: I would explicitly call out that the `.yml` files should provide detailed documentation for the individual steps, and that these are only broad examples of what the GitHub Actions can do.

Reply:
Done.

Reply:
Done.